src/software/ai/hl/stp/play/tactic_assignment_test.cpp

```diff
@@ -77,14 +78,18 @@ TEST_F(STPTacticAssignmentTest,
     move_tactic_2->updateControlParams(Point(1, 0), Angle::zero(), 0,
                                        TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT);
 
-    ConstTacticVector tactics = {move_tactic_1, move_tactic_2};
+    TacticVector tactics = {move_tactic_1, move_tactic_2};
 
-    auto asst = stp.assignRobotsToTactics({tactics}, world, true);
+    auto robot_navigation_obstacle_config = ai_config.robot_navigation_obstacle_config();
+    GlobalPathPlannerFactory path_planner_factory(robot_navigation_obstacle_config,
```

Comment:
on master, `GlobalPathPlannerFactory` doesn't exist anymore. We should update these tests according to `master`